### PR TITLE
fix(discord.js): url didnt support old docs

### PR DESCRIPTION
**Pull request by Kathund**
## Description 
Discord.js docs have moved to old.discordjs.dev while then remake the docs. I have added regex that supports the old docs url and the current docs url

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by theusaf** [comment]
Please provide screenshots showing that it works on both the new and old docs.

**Comment by Kathund** [comment]
> Please provide screenshots showing that it works on both the new and old docs.

Update Docs (Dosent support showing what page ur on. I can add this you wanted)


Main Website


Old docs


Old docs looking at something

File: websites/D/Discord.js/metadata.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
 		"nl": "Discord.js is een krachtige node.js module die je toelaat om zeer eenvoudig te interageren met de Discord API."
 	},
 	"url": "discord.js.org",
+	"regExp": "([a-z0-9-]+[.])*(discord[.]js.org|old[.]discordjs.dev)[/]",
 	"version": "1.2.19",
```
Comment (plskz) [inline websites/D/Discord.js/metadata.json:15]: bump version

```suggestion
	"version": "1.2.20",
```

**Comment by theusaf** [comment]
> Update Docs (Dosent support showing what page ur on. I can add this you wanted)

This would be helpful.

**Comment by Kathund** [comment]
> > Update Docs (Dosent support showing what page ur on. I can add this you wanted)
> 
> This would be helpful.

@theusaf How is this

https://github.com/PreMiD/Presences/assets/55346310/a3a8a4a2-b102-4d3e-9d29-e4a3bb91898d

File: websites/D/Discord.js/presence.ts [hunk 2]
```diff
@@ -7,51 +7,77 @@ presence.on("UpdateData", () => {
...
-            }`
-          : ""
-      }`;
+    if (route.length === 1 || route[1] === "")
+        data.details = "Browsing the main page...";
+      else if (route[1] === "docs") {
+        [, , data.smallImageKey] = route;
+        // data.smallImageText = `${route[2].replace(/^[a-z]/i, (c) =>
+        //   c.toUpperCase()
+        // )} - ${Number(route[3][0]) ? `v${route[3]}` : `${route[3]}`}`;
```
Comment (theusaf) [inline websites/D/Discord.js/presence.ts:40]:
```suggestion
```

File: websites/D/Discord.js/presence.ts [hunk 3]
```diff
@@ -7,51 +7,77 @@ presence.on("UpdateData", () => {
   const data: PresenceData = {
       largeImageKey: "https://cdn.rcd.gg/PreMiD/websites/D/Discord.js/assets/logo.png",
       startTimestamp: browsingStamp
-    },
+  }
+  var route = document.location.pathname.split("/");
```
Comment (theusaf) [inline websites/D/Discord.js/presence.ts:11]:
```suggestion
  let route = document.location.pathname.split("/");
```

**Comment by Kathund** [comment]
> The demo looks good! However, make sure to run the formatter (`npm run format`), the files are incorrectly formatted, although I'm unsure why the automated checks didn't catch that.

i ran npm format and it crashed my pc :skull: but i will look into it since its an issue on my end

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**